extension/runner_util/inputs.cpp

```diff
@@ -78,7 +78,40 @@ Result<BufferCleanup> prepare_input_tensors(
       continue;
     }
     if (tag.get() != Tag::Tensor) {
-      ET_LOG(Debug, "Skipping non-tensor input %zu", i);
+      if (!hard_code_inputs_to_ones) {
+        Error err = Error::Ok;
```

Comment:
do you want to add tests for this? Perhaps here - extension/runner_util/test/inputs_test.cpp